source/renderer/UiRenderer.cpp

```diff
@@ -50,7 +50,7 @@ void UiRenderer::render() {
   Text version;
   version.font     = "Pacaya";
   version.size     = 0.5f;
-  version.content  = std::string("Early testing build: ")  + GAME_VERSION;
+  version.content  = std::string("Early testing build: ") + "development 0.4.0-180-g3805388";
```

Comment:
GAME_VERSION is saved in file Version.hpp and this file is regenerated with cmake in each build